### DISCUSSION TITLE
About ENV_VARS

### DISCUSSION THREAD

**Discussion by cafkafk** [eza-community/eza#110] [Development]
Currently, the environment variables retain their old name, EXA_<something>.

If no one sees an issue with this, I'll change it to EZA_<something> in the foreseable future. Zetta wil probably remain backwards compatible.

**Comment by sbatial** [comment]
Voting to keep both (at least for some time), if it isn't a MAJOR pain.
Just because it would be an annoying break imo

**Comment by cafkafk** [comment]
> Voting to keep both (at least for some time), if it isn't a MAJOR pain. Just because it would be an annoying break imo

It really isn't any major pain to me, I don't use them  ¯\_(ツ)_/¯

Keeping both seem like a fine enough solution for now.
